contents/docs/tutorials/nuxt-js-integration.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Tracking Nuxt.js apps
+sidebar: Docs
+showTitle: true
+---
+
+If you are using Nuxt.js and want to track your application using PostHog this tutorial might help you out. 
+
+It will guide you through an example integration of PostHog using Nuxt.js plugins. 
+
+### Is This Tutorial For Me?
+
+This tutorial is aimed at Nuxt.js users which run Nuxt in `spa` or `universal` mode. 
+We are going to look at some minimal example code which should get you started quickly and provide a base for further customization.
+
+### Pre-Requisites
+
+To follow this tutorial along, you need to:
+
+1. Have [deployed PostHog](/docs/deployment).
+2. Have a running Nuxt.js application
+
+### Minmal Example
+
+We are going to implement PostHog as a [Nuxt.js plugin](https://nuxtjs.org/docs/2.x/directory-structure/plugins) which gives us the possibility to inject
+the posthog object and make it available across our application.
+
+The first thing you want to do is to install the [posthog-js library]((/docs/integrations/js-integration)) in your project - so add it using your package manager:
+
+```
+yarn add posthog-js
+```
+
+or
+
+```
+npm install --save posthog-js
+```
+
+After that we want to create a plugin in `plugins/posthog/index.js`
+
+```javascript
+import posthog from 'posthog-js'
+import Vue from 'vue'
+
+export default function({ app: { router } }, inject) {
+  /* Init PostHog
+  Note: In a production setup you don't want to have your secrets here but rather
+  use environment variables for it
+  */
+  posthog.init('POSTHOG_API_KEY', {
```

Comment:
RE the comment: This "API key" is not actually a secret, so it can be here no problem

Reply:
Ah nice. Then I will adjust the comment :)